ui/prometheus-plugin/src/plugins/prometheus-variables.tsx

```diff
@@ -67,6 +67,7 @@ function PrometheusLabelValuesVariableEditor(props: OptionsEditorProps<Prometheu
           datasourcePluginKind="PrometheusDatasource"
           value={selectedDatasource}
           onChange={handleDatasourceChange}
+          readOnly={props.isReadonly}
```

Comment:
Could be good to have prop named isReadonly too for consistency or something more descriptive like isDisabled if that’s what it is being used for

Reply:
DatasourceSelect is extending Mui Select props, Should I omit `readOnly` and create an alias?

Reply:
Oh I see my bad, yeah this fine!